e2e/tests/api-driven/src/globalHelpers.ts

```diff
@@ -10,6 +10,7 @@ export function createTeam(
       slug: "E2E",
       submissionEmail: TEST_EMAIL,
       homepage: "planx.uk",
+      referenceCode: "ABCD",
```

Comment:
This is a mandatory field to generate a valid payload. `planx-core` now makes this required with this change - https://github.com/theopensystemslab/planx-core/pull/337